pkg/exposer/pod_volume.go

```diff
@@ -394,6 +398,9 @@ func (e *podVolumeExposer) createHostingPod(ctx context.Context, ownerObject cor
 					Env:             podInfo.env,
 					EnvFrom:         podInfo.envFrom,
 					Resources:       resources,
+					SecurityContext: &corev1api.SecurityContext{
```

Comment:
Windows containers don't support this. So need to check the nodeOS and set this for non-Windows only.